src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import Label from "../Label/Label";
+
+export interface InputProps {
+  /** An optional text label for the radio button */
+  labelText?: string;
+  /** Additional attributes to pass to the <input> tag */
```

Comment:
Wrap `<input>` in backticks. It renders weirdly without in storybook:

<img width="240" alt="Screen Shot 2021-04-09 at 12 06 15 PM" src="https://user-images.githubusercontent.com/1280564/114208903-023b8d00-992c-11eb-8583-f13b78ad0064.png">

Reply:
Fixed!